packages/core/src/connectivity/scp-cf/destination/destination-from-service.ts

```diff
@@ -380,8 +380,8 @@ class DestinationFromServiceRetriever {
 
   private async addClientCertAuth(): Promise<Destination> {
     const accessToken = await serviceToken('destination', {
-      userJwt: this.decodedUserJwt,
-      ...this.options
+      ...this.options,
+      userJwt: this.decodedUserJwt
```

Comment:
Does this mean, that if I set `iss` in my options and this is also set in the JWT, it will be ignored?
Is this what we want? I see advantages/disadvantages on both sides.

If we leave it as is, we should add a compatibility note.

Reply:
no in the initialitaion a JWT is build from the iss if present and then used. It is not validated like a JWT>